webapp/components/test-runs-query.html

```diff
@@ -137,7 +91,7 @@
           if (maxCount) {
             params['max-count'] = maxCount;
           }
-          if (this.aligned) {
+          if (aligned) {
```

Comment:
I have a small feature request, the change wouldn't be here, but I also don't know where. For params that have a boolean value, would it not be nicer to just append `&aligned` instead of `&aliged=true`? C.f. `&diff`. Came here because I just found myself writing a URL by hand and was not at all tempted to append `=true`.

Reply:
Noted. Will play around in a follow-up PR.